src/auth/oauth2client.ts

```diff
@@ -936,4 +932,16 @@ export class OAuth2Client extends AuthClient {
     const buffer = new Buffer(b64String, 'base64');
     return buffer.toString('utf8');
   }
+
+  /**
+   * Returns true iff a token is expired or will expire within
```

Comment:
Typo `iff`

Reply:
Meant "if and only if", but changed to "if".